### ISSUE TITLE
Will you please update the guide for us beginners?

### ISSUE THREAD

**Issue by chn-cutelotus** [PistonDevelopers/conrod#1358]
I'm searching for UI frameworks for Rust programming. To my disappointment, none of them supports Chinese characters except Conrod. I'm very interested in this project. However, it seems it's been a long time since the last guide updated.

**Comment by guygastineau** [comment]
I agree.  I have been messing with Druid from the xi-editor project, but `conrod` is very enticing.  Unfortunately, we all experience that sinking feeling where the Guide ceases to exist right at the content we want ;)

For what it is worth, I plan on building some internal apps for my work with conrod.  If it is welcome I will take a stab at a PR for the `Let's Create a GUI` section of the guide, but I have to figure out how to use it first anyway.

Hopefully, someone already involved in this project will share some insight with us in this issue <3

**Comment by guygastineau** [comment]
By the way, @chn-cutelotus , here are some projects by other people on GH that help show how to get started:
(https://github.com/Finnerale/conrod-boilerplate)
(https://github.com/zen3ger/conrod-guessing_game)
(https://github.com/Ophirr33/conrod-hello-world/blob/master/src/main.rs)
